x-pack/plugins/ingest_manager/server/routes/data_streams/handlers.ts

```diff
@@ -110,11 +104,15 @@ export const getListHandler: RequestHandler = async (context, request, response)
         dataset: { buckets: datasetBuckets },
         namespace: { buckets: namespaceBuckets },
         type: { buckets: typeBuckets },
-        package: { buckets: packageBuckets },
         last_activity: { value_as_string: lastActivity },
       } = result;
 
-      const pkg = packageBuckets.length ? packageBuckets[0].key : '';
+      // We don't have a reliable way to associate index with package ID, so
```

Comment:
@skh you might have implemented some magic here for linking to the dashboards?

Reply:
The magic is on the next line and isn't affected :)

Reply:
Ok, I haven't checked the code from @skh but if I got your comment right, she does the linking the same way?